aslprep/workflows/asl/apply.py

```diff
@@ -148,7 +148,7 @@ def init_asl_cifti_resample_wf(
 
         workflow.connect([
             (inputnode, goodvoxels_bold_mask_wf, [
-                ("asl_file", "inputnode.bold_file"),
+                ("asl_anat", "inputnode.bold_file"),
```

Comment:
I was feeding in the minimally-preprocessed aslref-space ASL file here, so the goodvoxels mask was in a different space and resolution than the anatomical-space data meant to be put in the CIFTI.

Reply:
This is what was causing #360 I believe.